src/Framework/TestSuiteIterator.php

```diff
@@ -14,21 +14,18 @@
 /**
  * Iterator for test suites.
  */
-class TestSuiteIterator implements RecursiveIterator
+final class TestSuiteIterator implements RecursiveIterator
```

Comment:
btw, this is a  BC breaker for all of us that were using this class. (class is not marked as `@internal`)

Reply:
In the end I didn't need to change many of the existing internals as the code moved to TestSuiteSorter. Extensions support is coming, I tried that first :)